algebird-core/src/main/scala/com/twitter/algebird/package.scala

```diff
@@ -17,6 +17,8 @@
 package com.twitter
 
 package object algebird {
+  type Seq[+T] = scala.collection.Seq[T]
+  type IndexedSeq[+T] = scala.collection.IndexedSeq[T]
```

Comment:
is there no way we can't use `collection.immutable` here? Why do we need these aliases?